interfaces/builtin/utils_test.go

```diff
@@ -142,6 +142,42 @@ func (s *utilsSuite) TestSlotLabelExpr(c *C) {
 	c.Check(label, Equals, `"snap.test-snap.*"`)
 }
 
+func (s *utilsSuite) TestAareExclusivePatterns(c *C) {
```

Comment:
I think this test needs a few more variants of "scary" looking strings to ensure we do the right thing (but maybe some of these examples produce invalid apparmor rules or are otherwise unsupported by apparmor's globbing), examples off the top of my head include:

* something with a `.` in it
* something with a ` ` (space) in it
* something with a `,` in it
* something with a `[` and a `]` in it

Reply:
I did these and more. Thanks.